web/docs/guides/integrations/clerk.mdx

```diff
@@ -0,0 +1,184 @@
+---
+id: clerk
+title: 'Clerk'
+description: 'This guide explains how to connect your Supabase database with Clerk, a powerful authentication provider built for the modern web.'
+---
+
+This guide explains how to connect your Supabase database with [Clerk](https://clerk.dev), a powerful authentication provider built for the modern web.
```

Comment:
```suggestion
This guide explains how to connect your Supabase database with [Clerk](https://clerk.dev), an authentication provider built for the modern web.
```

Since these are Docs, it's probably better to remove any "marketing speak".